metaflow/_vendor/click/_termui_impl.py

```diff
@@ -405,7 +405,8 @@ def _tempfilepager(generator, cmd, color):
     """Page through text by invoking a program on a temporary file."""
     import tempfile
 
-    filename = tempfile.mktemp()
```

Comment:
@fatmo666 this file is vendored as is from click. to fix this, you would need to update the vendored version for click.